### PR TITLE
Feat add operator

### PR DESCRIPTION
**Pull request by Sijoma**
Operator scaffold stuff & api.

Example: camunda-scaling-operator/config/samples/camunda_v1alpha1_zeebeautoscaler.yaml
```yaml
apiVersion: camunda.sijoma.dev/v1alpha1
kind: ZeebeAutoscaler
metadata:
  name: zeebeautoscaler-sample
spec:
  minReplicas: 1
  maxReplicas: 2
  zeebeRef:
    name: my-zeebe
```

### CODE REVIEW

**Comment by Sijoma** [comment]
How it potential can work now:

```bash
make setup-dev
cd camunda-scaling-operator
make deploy-test-env
kubectl get pods -n camunda-scaling-operator-system
```

Probably it breaks as some dependencies might not directly work. But who knows :D.

File: camunda-scaling-operator/Dockerfile
Comment (npepinpe) [inline camunda-scaling-operator/Dockerfile:1]: :wrench: No build stage? :crying_cat_face:

File: camunda-scaling-operator/Makefile
Comment (npepinpe) [inline camunda-scaling-operator/Makefile:1]: I assume you just wrote this from scratch right now :wink:

File: camunda-scaling-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# camunda-scaling-operator
+Scale camunda like no other. ⚡️
+
+## Description
+Scale scale scale!
```
Comment (npepinpe) [inline camunda-scaling-operator/README.md:5]: image

**Review by npepinpe — APPROVED** [review]

**Pull request merged**